feat: add componentProviders for component-level dependency injection (#16)
hirezio/vitest-browser-angular: README.md

});
```

## Component Providers

> If you need to override providers defined on the component decorator, you can use the `componentProviders` option:

Reviewer: nit: Maybe we should make it clear that this will not replace the whole providers array of the component.

```suggestion
If you need to add or override [component providers](https://angular.dev/guide/di/defining-dependency-providers#component-or-directive-providers), you can use the `componentProviders` option.
```
